### DISCUSSION TITLE
Why are the Handicap odds inverted on the away side?

### DISCUSSION THREAD

**Discussion by kgravenreuth** [cloudbet/docs#26] [Q&A]
For handicap markets, there is special handling which should be noted. This is also mentioned in our [Cloudbet API Documentation](https://www.cloudbet.com/api/#/API-Version-2.0/GetEvent?utm_source=github.com&utm_medium=community&utm_campaign=axis), while we are adding this explanation here along with an example screenshot for added clarity.

For handicap markets a `line` is identified by the same `params`. `Selections` can be grouped by `marketKey` and `params` alone. `Home` and `away` outcomes then have the same `marketURL` for the same handicap lines. 

**The handicap value is dictated by the home team value and inverted on the away side**.

Example (taken from https://github.com/Cloudbet/docs/discussions/24 )

The API returns (part of the API response):

```
Outcome: "away",
Params: "handicap=0.5",
Price: 3.3
```

This means that the corresponding asian handicap odds displayed on our frontend is the one highlighted below (as the handicap value is dictated by the home team value and inverted on the away side):

![frontend](https://user-images.githubusercontent.com/21211077/162711555-e53fd847-3095-434e-98e7-bedd8c16a0a1.png)

**Comment by wajdi-20** [comment]
thanks very much for the clarification
